code/modules/fishing/fishing_rod.dm

```diff
@@ -66,9 +63,10 @@
 	return NONE
 
 /obj/item/fishing_rod/Destroy(force)
-	. = ..()
-	//Remove any leftover fishing lines
-	QDEL_LIST(fishing_lines)
+	bait = null
+	line = null
+	hook = null
```

Comment:
shouldn't you be qdel'ing these

Reply:
Because they're going to be qdeleted anyway, being in its contents.

Reply:
I'd prefer if(!isnull(x))

Reply:
qdel null will end up qdeling null if the var is already null

Reply:
qdeling null isn't an issue last I checked, it just early returns.